server/tournaments/index.ts

```diff
@@ -388,6 +388,11 @@ export class Tournament extends Rooms.RoomGame<TournamentPlayer> {
 			return;
 		}
 
+		if (user.namelocked) {
+			output.sendReply('|tournament|error|UserNameLocked');
```

Comment:
```suggestion
			output.sendReply('|tournament|error|UserNameLocked');
```
I don't think this needs an entirely new case just to shorthand something.

```suggestion
			output.popup("You are banned from entering tournaments.");
```
This would be preferable, but if we're really set on it being in the tournament popup: 
```suggestion
			output.sendReply('|tournament|error|Banned');
```
Adding a new case for the same message doesn't make much sense.

Reply:
if we chose the latter case, shouldn't [this line](https://github.com/smogon/pokemon-showdown/blob/b0beb838254601c87342fa17e7e12eabb66ff9ce/server/tournaments/index.ts#L402) also be popup?

Reply:
It's fine either way /shrug. 
Changing it to just `Banned` is probably easier for now, though. Tours needs to not hardcode so many messages in the client, but that's a future project probably.

Reply:
Ok, I'm changing it to `Banned`, I hope the developers which refactor that will do good